src/app/components/Sidebar/index.tsx

```diff
@@ -196,11 +204,12 @@ const SidebarFooter = (props: SidebarFooterProps) => {
           </Menu>
         </Box>
       </SidebarTooltip>
-      <Box align="center" pad="small">
-        <a href="https://github.com/oasisprotocol/oasis-wallet-web" target="_blank" rel="noopener">
-          <Github />
-        </a>
-      </Box>
+      <SidebarButton
+        icon={<Github />}
+        label="GitHub"
+        route="https://github.com/oasisprotocol/oasis-wallet-web"
+        newTab
```

Comment:
By "consistent," you mean to re-use the same component right? In which, 👍

Reply:
(yes, but also having a label/tooltip)

Reply:
Neato.